tests/test_modules.py

```diff
@@ -206,7 +222,7 @@ def test_get_requirements(
 
 @pytest.mark.parametrize("editable", [True, False])
 def test_module_v2_source_get_installed_module_editable(
-    snippetcompiler,
+    snippetcompiler_clean,
```

Comment:
This test installs test packages in the snippetcompiler venv. Not using a clean snippetcompiler causes instability issues for other tests.

Reply:
I would add that comment to the code